ProcessMaker/Traits/MakeHttpRequests.php

```diff
@@ -53,6 +53,12 @@ public function request(array $data = [], array $config = [])
             $url .= $separator . $config['queryString'];
         }
 
+        // If exists a filter string in the call, add it to the URL
+        if (array_key_exists('filter', $config)) {
+            $separator = strpos($url, '?') ? '&' : '?';
+            $url .= $separator . 'filter='. urlencode($config['filter']);
```

Comment:
Don't we already do this by adding a filter={filter} in the End Point URL in the Data Source?

Reply:
the default data connector created for collection does not includes `filter` parameter
That's why it was added dynamicaly like `pmql` parameter
![image](https://user-images.githubusercontent.com/8028650/91068782-c228d380-e602-11ea-8662-c8dc46426f71.png)